CONTRIBUTING.md

```diff
@@ -1,3 +1,63 @@
----
-layout: page
----
+# Hilf mit unsere Seite besser zu machen!
+
+Wir freuen uns, dass du dir überlegst, dich an diesem Repository zu beteiligen.
+Zunächst solltest du wissen, dass du, wenn du zu diesem Repository beitragen
+möchtest, unseren
+[Lizenzbestimmungen](/LICENSE.txt)
```

Comment:
Die Datei im Repo heißt aktuell `LICENSE`, wäre wahrscheinlich am geschicktesten, wenn wir die in `LICENSE.txt` umbenennen, dann ist auch direkt der Dateityp erkennbar.